lib/bambora/bank/payment_profile_resource.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Bank
+    ##
+    # For making requests to the /scripts/payment_profiles.asp endpoint
+    class PaymentProfileResource
+      DEFAULT_VERSION = 1.0
+      attr_reader :client, :api_key, :sub_path, :version
+
+      ##
+      # Instantiate an interface to make requests against Bambora's Profiles API.
+      #
+      # @example
+      #
+      #   client = Bambora::WWWFormClient(base_url: '...', api_key: '...', merchant_id: '...')
+      #   profiles = Bambora::Bank::PaymentProfileResource(client: client)
+      #
+      #   # Start making requests ...
+      #
+      # @param client [Bambora::JSONClient] An instance of Bambora::JSONClient, used to make network requests.
+      # @param api_key [String] An API key for this endpoint. This is also known as the "Pass Code"
+      # @param version [String] The Service Version you are requesting from the server.
+      def initialize(client:, api_key:)
+        @client = client
+        @api_key = api_key
+        @sub_path = '/scripts/payment_profiles.asp'
+        @version = version || DEFAULT_VERSION
+      end
+
+      ##
+      # Create a Bank Payment Profile
+      #
+      # @example
+      #  data = {
+      #   customer_code: '1234',
+      #   bank_account_type: 'CA',
+      #   account_holder: 'All-Maudra Mayrin',
+      #   institution_number: '123',
+      #   branch_number: '12345',
+      #   account_number: '123456789',
+      #   name: 'Hup Podling',
+      #   email_address: 'Brea Princess of Vapra',
+      #   phone_number: '1231231234',
+      #   address_1: 'The Castle',
+      #   city: "Ha'rar",
+      #   postal_code: 'H0H 0H0',
+      #   province: 'Vapra',
+      #   country: 'Thra',
+      #   sub_merchant_id: '1',
+      #  }
+      #
+      #  payment_profile_resource.create(data)
+      #
+      #  @params profile_data [Hash] with values as noted in the example.
+      def create(profile_data)
+        client.post(path: sub_path, body: payment_profile_body(profile_data))
```

Comment:
The example above is missing an `operationType`, which in the case of creation should be `N`. Could this be hard-coded into the implementation?

Reply:
Ah, yes yes. This should be hard-coded into the implementation!